CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## 0.17.11-dev
+  * Add support for Flutter locale split.
+
+## 0.17.10+2
+  * Allow null safe code when parsing.
+
+## 0.17.10+1
```

Comment:
This was the last version that was published -> https://pub.dev/packages/intl_translation/versions

Might want to clean-up the changelog to remove the 0.17.10+2 release and roll that bit into 0.17.11